test/users/get_users.test.js

```diff
@@ -0,0 +1,49 @@
+const request = require('supertest');
+const helper = require('../../app/helpers/users');
+const { createUser } = require('../factory/factory_users');
+
+const mockToken = jest.spyOn(helper, 'decodeToken');
+const app = require('../../app');
+
+describe('Tests to get all users', () => {
+  const user = {
+    name: 'pepito',
+    last_name: 'Perez',
+    mail: 'pepito@wolox.co',
+    pass: 'PassPepit0'
+  };
+
+  const paginationQuery = {
+    page: '1',
+    zise: '1'
+  };
+
+  afterEach(() => {
+    mockToken.mockRestore();
+  });
```

Comment:
For next time, you could use mockImplementationOnce to avoid restoring it after each test. It is a matter of opinion anyway.